### PR TITLE
Merge release-5.2.0 into release-5.3.0

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -38,13 +38,16 @@ const routes: Routes =  [
...
+    path: 'players/interactive/:id', component: InteractivePlayerComponent,
   },
   {
     path: 'players/collection', component: CollectionPlayerComponent,
   },
   {
     path: 'players/:mimeType/:id', component: PlayersComponent,
+  },
+  {
+    path: 'players/interactive', component: InteractivePlayerComponent,
```
Comment (vinukumar-vs) [inline src/app/app-routing.module.ts:50]: Why are we adding both here for InteractivePlayerComponent?

Comment (vaibhavbhuva) [reply src/app/app-routing.module.ts:50]: As of now, InteractivePlayerComponent is not part of comman player component. hence we are handling separately and added new route for this.

Comment (vinukumar-vs) [reply src/app/app-routing.module.ts:50]: path: 'players/interactive', component: InteractivePlayerComponent,

path: 'players/interactive/:id', component: InteractivePlayerComponent,

Both the paths launching the same Component.

Comment (vaibhavbhuva) [reply src/app/app-routing.module.ts:50]: `path: 'players/interactive', component: InteractivePlayerComponent,` 
The above path is added for loading default content from the player list.

`path: 'players/interactive/:id', component: InteractivePlayerComponent,`
This route added for loading dynamic content from the content list page (Review and publish the content)

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**